index.js

```diff
@@ -88,6 +88,10 @@ export default class PayWithAmazon extends Emitter {
     }
   }
 
+  get requiresConsent() {
+    return !!this.config.consent;
+  }
+
```

Comment:
**comment**: my read on this is that we must pass a truthy value on the config.consent for the customer to see the consent checkbox option in their checkout flow.

Reply:
check this [comment](https://github.com/recurly/pay-with-amazon/pull/39#discussion_r891451928)